robot_sf/sim/fast_pysf_wrapper.py

```
@@ -0,0 +1,239 @@
+    ) -> None:
+        """Precompute and store a sampled force grid under the given `name`.
+
+        The cached grid can later be interpolated with `interpolate_force`.
+        If a cache with the same name exists it will be overwritten.
+        """
+        xs_arr = np.asarray(xs, dtype=float)
+        ys_arr = np.asarray(ys, dtype=float)
+        field = self.get_force_field(xs_arr, ys_arr, **kwargs)
+        self._force_grid_caches[name] = {"xs": xs_arr, "ys": ys_arr, "field": field}
+
+    def interpolate_force(self, x: float, y: float, name: str = "default") -> np.ndarray:
```

Pylint (3.3.7)

[refactor] Too many local variables (22/15)

(R0914)